docs/tutorials/TracingCode.md

```diff
@@ -189,22 +189,22 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
    @Override
    public CommandResult execute(Model model) throws CommandException {
        ...
-       Person personToEdit = lastShownList.get(index.getZeroBased());
-       Person editedPerson = createEditedPerson(personToEdit, editPersonDescriptor);
-       if (!personToEdit.isSamePerson(editedPerson) && model.hasPerson(editedPerson)) {
+       Person questionToEdit = lastShownList.get(index.getZeroBased());
+       Person editedQuestion = createEditedPerson(questionToEdit, editPersonDescriptor);
```

Comment:
Should these be `Question questionToEdit` and `Question editedQuestion`?

Reply:
I think the changes to the docs can be done later on? There are a lot of changes to be made

Reply:
I changed this part but the other docs will still need to be updated